### PR TITLE
Update sophora-schema-docs

### PR DESCRIPTION
**Pull request by blizzy78**
This PR updates the chart for Sophora Schema Docs:

- Removes support for static mode. This isn't really useful in a Kubernetes environment and is more intended for local usage.
- Adds liveness and readiness probes.
- Some general cleanup in a few places.

Please note that I've bumped up the chart version to 2.0.0 for good measure.

### CODE REVIEW

File: charts/sophora-schema-docs/templates/deployment.yaml [hunk 1]
```diff
@@ -22,47 +22,54 @@ spec:
...
-            - mountPath: /data
-          {{else}}
-            - mountPath: /config
-          {{end}}
-              name: config
-            - mountPath: /out
-              name: output-folder
+            - name: config
+              # 4.0.0 is the only version that uses WORKDIR /data
+              {{if eq (.Values.image.tag | default .Chart.AppVersion) "4.0.0" }}
```
Comment (tom-schoener) [inline charts/sophora-schema-docs/templates/deployment.yaml:32]: By adding a `-` after `{{` you remove unnecessary whitespace after rendering the yaml file. Not super important but good practice.

File: charts/sophora-schema-docs/test-values.yaml
Comment (tom-schoener) [inline charts/sophora-schema-docs/test-values.yaml:1]: The `test-values.yaml` file is useful to check if the chart renders with some basic settings albeit not the most precise test. You should restore the file imo.

**Review by tom-schoener — CHANGES_REQUESTED** [review]
Changes look good for the most part. After restoring the test file this PR is ready for merging

File: charts/sophora-schema-docs/templates/deployment.yaml (hunk 1 above)
Comment (blizzy78) [reply charts/sophora-schema-docs/templates/deployment.yaml]: I've made modifications to remove said whitespace.

File: charts/sophora-schema-docs/test-values.yaml
Comment (blizzy78) [reply charts/sophora-schema-docs/test-values.yaml:1]: I have restored the file.

**Review by tom-schoener — APPROVED** [review]
